### PR TITLE
Upgrade to play 2.8.13

### PR DESCRIPTION
**Pull request by tomrf1**
The dependency `akka-http-core` has been marked by snyk as high severity.

Updates to scaffeine/play-googleauth necessary because of a `scala-java8-compat_2.13` version incompatibility

### CODE REVIEW

File: support-payment-api/src/main/scala/actions/RateLimitingAction.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ import play.api.libs.circe.Circe
   * from the backend, not the browser.
   */
 
-case class RateLimitingSettings(maxRequests: Int, interval: Duration)
+case class RateLimitingSettings(maxRequests: Int, interval: FiniteDuration)
```
Comment (tomrf1) [inline support-payment-api/src/main/scala/actions/RateLimitingAction.scala:22]: this is a small change in scaffeine

File: support-services/src/main/scala/com/gu/support/redemption/corporate/DynamoTableAsync.scala [hunk 2]
```diff
@@ -10,8 +10,8 @@ import software.amazon.awssdk.regions.Region
 import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
 import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, GetItemRequest, UpdateItemRequest}
 
-import scala.collection.JavaConverters._
-import scala.compat.java8.FutureConverters._
+import scala.jdk.CollectionConverters._
+import scala.jdk.FutureConverters._
```
Comment (tomrf1) [inline support-services/src/main/scala/com/gu/support/redemption/corporate/DynamoTableAsync.scala:14]: this is how we convert from java `CompletableFuture` to scala `Future` in scala 2.13

File: support-services/build.sbt [hunk 3]
```diff
@@ -6,7 +6,6 @@ description := "Scala library to provide shared services to Guardian Support pro
 
 libraryDependencies ++= Seq(
   "software.amazon.awssdk" % "dynamodb" % awsClientVersion2,
-  "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.1",
```
Comment (tomrf1) [inline support-services/build.sbt:9]: no longer needed and incompatible with deps

File: support-frontend/build.sbt [hunk 4]
```diff
@@ -31,7 +31,7 @@ libraryDependencies ++= Seq(
   "com.gu" %% "identity-test-users" % "0.8",
   "com.google.guava" % "guava" % "29.0-jre",
   "io.lemonlabs" %% "scala-uri" % scalaUriVersion,
-  "com.gu.play-googleauth" %% "play-v27" % "2.1.0",
+  "com.gu.play-googleauth" %% "play-v27" % "2.2.2",
```
Comment (johnduffell) [inline support-frontend/build.sbt:34]: existing, but that's a bit odd, why play-v27?

File: support-services/build.sbt (hunk 3 above)
Comment (johnduffell) [reply support-services/build.sbt:9]: nice well done for deleting it

**Review by johnduffell — APPROVED** [review]
nice thanks for bumping play in both those apps 👏

File: support-frontend/build.sbt (hunk 4 above)
Comment (tomrf1) [reply support-frontend/build.sbt:34]: oh I missed that! I'm surprised it even works

**Pull request merged**